frontend/src/app/main/ui/dashboard/team.cljs

```diff
@@ -571,6 +577,229 @@
+       [:span.icon.success i/msg-success]
+       [:span.icon.failure i/msg-warning])])
+
+  (mf/defc webhook-item
+    {::mf/wrap [mf/memo]}
+    [{:keys [webhook] :as props}]
+    (let [on-edit #(st/emit! (modal/show :webhook {:webhook webhook}))
+          error-code (:error-code webhook)
+          extract-status
+          (fn [error-code]
+            (let [status (-> error-code
+                             (str/split  "-")
+                             last)]
+              (if (js/isNaN status)
+                ""
+                (js/parseInt status))))
+          delete-fn
+          (fn []
+            (let [params {:id (:id webhook)}
+                  mdata  {:on-success #(st/emit! (dd/fetch-team-webhooks))}]
+              (st/emit! (dd/delete-team-webhook (with-meta params mdata)))))
+          on-delete #(st/emit! (modal/show
+                                {:type :confirm
+                                 :title (tr "modals.delete-webhook.title")
+                                 :message (tr "modals.delete-webhook.message")
+                                 :accept-label (tr "modals.delete-webhook.accept")
+                                 :on-accept delete-fn}))]
+      [:div.table-row
+       [:div.table-field.last-delivery
+        [:div.icon-container (if (nil? error-code)
+                               [:& last-delivery-icon {:success? true :text (tr "dashboard.webhooks.last-delivery.success")}]
+                               [:& last-delivery-icon {:success? false :text
+                                                       (cond
+                                                         (= error-code "ssl-validation") (tr "dashboard.webhooks.last-delivery.error.ssl-validation")
+                                                         (str/starts-with? error-code "unexpected-status")
+                                                         (tr "dashboard.webhooks.last-delivery.error.unexpected-status" (extract-status error-code))
+                                                         :else (tr "dashboard.webhooks.last-delivery.error.unknown"))}])]]
+       [:div.table-field.uri
+        [:div (:uri webhook)]]
+       [:div.table-field.active
+        [:div (if (:is-active webhook) (tr "labels.active") (tr "labels.inactive"))]]
+       [:div.table-field.actions
+        [:& webhook-actions {:on-edit on-edit
+                             :on-delete on-delete}]]]))
+
+
+(mf/defc webhooks-list
+  [{:keys [webhooks] :as props}]
+  [:div.dashboard-table
+   [:div.table-rows
+    (for [webhook webhooks]
+      [:& webhook-item {:webhook webhook :key (:id webhook)}])]])
+
+(mf/defc team-webhooks-page
+  [{:keys [team] :as props}]
+  (let [webhooks (mf/deref refs/dashboard-team-webhooks)]
+
+    (mf/with-effect [team]
+      (dom/set-html-title
+       (tr "title.team-webhooks"
+           (if (:is-default team)
+             (tr "dashboard.your-penpot")
+             (:name team)))))
+
+    (mf/with-effect
```

Comment:
I gess the empty `[]` is missing after `with-effect` indicating no dependencies

Reply:
Fixed